WorkloadManagementSystem/JobWrapper/JobWrapperTemplate.py

```diff
@@ -136,7 +138,10 @@ def execute ( arguments ):
 ##########################################################
 ret = -3
 try:
-  jobArgs = eval( """@JOBARGS@""" )
+  jobArgs = None
+  jsonFileName = os.path.realpath( __file__ ) + '.json'
+  with open( jsonFileName, 'r' ) as f:
+    jobArgs = json.loads( f.readlines()[0] )
   ret = execute( jobArgs )
```

Comment:
maybe a test that jobArgs is not None is also a good idea, otherwise we will just get an exception that NoneType does not have [] operator